lmnet/lmnet/networks/quantize_param_init.py

```diff
@@ -0,0 +1,123 @@
+import tensorflow as tf
+import functools
+
+
+class QuantizeParamInit:
+    """QuantizeParamInit is a mixin creating a quantized network class more convenient by providing two functions.
+    1) __init___ : for initializing variables necessary for quantization, such as quantization function
+    2) _quantized_variable_getter : for quantizing weight variables in convolutional layers
+
+    This QuantizeParamInit class can be used for all kinds of task (classification, object detection, and segmentation).
+
+    How to use
+    1) Extend this mixin
+    2) Set the custom_getter variable to `self.custom_getter` if its base class does not set it yet
+       The setting is done by adding the base function defining custom_getter for tensorflow inside the class as follows
+             def base(self, images, is_training):
+                 with tf.variable_scope("", custom_getter=self.custom_getter):
+                     return super().base(images, is_training)
+
+    In the object construction step, users must pass
+       * arguments for the quantization (e.g. activation_quantizer) followed by
+       * arguments of the subtask class (e.g. weight_decay_rate)
```

Comment:
What is `subtask class` ? need to change.

Reply:
I mean the `network class` that doing the subtask(classification, object detection).
I changed `subtask class` to `network class`.